tests/GraphCompareTest.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+import os
+import json
+import string 
+import random
+from .constants.TestPayload import *
+from .constants.TestEnvironment import *
+
+
+from app import app
+
+CURRENT_DIRECTORY = os.path.dirname(__file__)
+
+
+
+
```

Comment:
remove useless whitespace here and in the rest of the file

Reply:
done